GEOSldas_App/util/postproc/ObsFcstAna_stats/user_config.py

```diff
@@ -0,0 +1,132 @@
+
+import sys;       sys.path.append('../../shared/python/')
+import warnings;  warnings.filterwarnings("ignore")
+import os
+
+import numpy as np
+
+from datetime               import datetime, timedelta
+from read_GEOSldas          import read_tilecoord, read_obs_param
+from postproc_ObsFcstAna    import postproc_ObsFcstAna
+
+def get_config():
+
+    # =========================================================================================
+    #
+    # User-defined inputs for post-processing of ObsFcstAna output
+
+    # Range of months to process:
+    
+    start_year  = 2015
+    start_month =    4
+    last_year   = 2016
+    last_month  =    4
+
+    # Sums or stats will be processed for exp_main:
+
+    exp_main = {
+        'expdir'        : '/discover/nobackup/projects/gmao/merra/iau/merra_land/SMAP_runs/SMAP_Nature_v11/',
+        'expid'         : 'DAv8_SMOSSMAP',
+        'exptag'        : 'DAMulti_SMAP', 
+        'domain'        : 'SMAP_EASEv2_M36_GLOBAL',
+        'da_t0'         : 3,                              # hour of first land analysis
+        'da_dt'         : 10800,                          # ObsFcstAna file interval in seconds
+        'species_list'  : [5,6,7,8],
+        'obsparam_time' : "20150401_0000"                 # time stamp of obsparam file (YYYYMMDD_HHMM)
+    }
+
+    # Optional experiment(s) can be added for cross masking or extracting obs from a different experiment.
+    #
+    # All optional experiments and the main experiment must have identical tilecoords.
+    # If the default "species" number/order do not match, set "species_list" accordingly to force a match.
+    # Output will be cross-masked between all specified experiments.    
+    
+    exp_sup1 = {
+        'expdir'        : '/discover/nobackup/projects/gmao/merra/iau/merra_land/SMAP_runs/SMAP_Nature_v11/',
+        'expid'         : 'DAv8_M36',
+        'exptag'        : 'DASMAP_SMAP', 
+        'domain'        : 'SMAP_EASEv2_M36_GLOBAL',
+        'species_list'  : [1,2,3,4],
+        'obsparam_time' : "20150401_0000"                 # time stamp of obsparam file (YYYYMMDD_HHMM)
+    }
+
+    # Convert experiments input to a list; first entry must be exp_main 
+
+    exp_list = [exp_main, exp_sup1]
+    
+    # Forecasts and analyses are always from the main experiment.
+    # Observations can be from experiment indicated by 'obs_from' index (0-based).
+    # The mostly likely use case for this is that _scaled_ observations from a DA experiment
+    #   are used to compute OmF etc diagnostics for a corresponding open loop experiment.
+
+    obs_from = 1              # 0-based index of exp in exp_list from which obs are extracted
```

Comment:
Just a thought -- would it be cleaner to have a new field "use_obs" as part of the exp_XXXX structure?  That would get rid of the additional variable.  We can add a check that exactly one exp in exp_list has use_obs=true, all others must be false.

Reply:
suggestion is implemented.